utils/preProcessing/stroopwafelInterface.py

```diff
@@ -234,10 +234,11 @@ def rejected_systems(locations, dimensions):
     # Over-ride with pythonSubmit parameters, if desired
```

Comment:
Change this comment to:
Over-ride with runSubmit + compasConfigDefault parameters, if desired

Same in line 22 of this file, where the comment "Include options from local pythonSubmit file" should be "Include options from local runSubmit + compasConfigDefault files".